CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 #### [Unreleased](https://github.com/hougesen/mdsf/compare/v0.2.6...HEAD)
 
+- feat(formatters): support superhtml [`#479`](https://github.com/hougesen/mdsf/pull/479)
```

Comment:
Why does the changelog of PR 480, contains the entry that could be in the 479 one?

And it's the same in 479, talking about things added in 478?

Is it because you don't have the PR id when you commit (obviously),but it voildbe updated with amend+ force merge once the PR is opened.

I'm asking because the history of the changelog is strange if you use git blame

Reply:
Yes, the changelog generator (https://github.com/CookPete/auto-changelog) uses commit data which means it is one commit behind.

For now it is just a part of my precommit command, but yes, it would be a lot clearer to append instead.